corehq/apps/zapier/urls.py

```diff
@@ -2,14 +2,18 @@
 
 from corehq.apps.api.urls import CommCareHqApi
 from corehq.apps.zapier.api.v0_5 import ZapierXFormInstanceResource, ZapierCustomFieldResource
+from corehq.apps.zapier.api.v0_5 import ZapierCustomTriggerFieldFormResource, ZapierCustomFieldCaseResource
 from corehq.apps.zapier.views import SubscribeView, UnsubscribeView
 
 hq_api = CommCareHqApi(api_name='v0.5')
 hq_api.register(ZapierXFormInstanceResource())
 hq_api.register(ZapierCustomFieldResource())
```

Comment:
Is this one still needed? This is the `Base` Resource so I don't think you should be able to access it through a url.